test.sh

```diff
@@ -12,7 +12,7 @@ trap 'err_report $LINENO' ERR
 grepOutput() {
     local output_file="command_output.tmp"
 
-    if ! "${@:1:(($# - 1))}" > $output_file
+    if ! "${@:1:(($# - 1))}" &> $output_file
```

Comment:
I suppose you tested this ? I would expect this syntax to daemonize the process, not redirect `stderr`.
The syntax I expected was `2>&1 >$output_file`

Reply:
`&>` captures both `stdout` and `stderr`, see the [bash reference manual](https://www.gnu.org/software/bash/manual/html_node/Redirections.html), 3.6.4.

Reply:
Btw, this change is required to fix this test:

https://github.com/dlang-tour/core-exec/blob/ac80fa8c937f71788cbe27025419d95af308a706/test.sh#L127-L133